fedimint-server/src/config.rs

```diff
@@ -226,6 +256,7 @@ impl ServerConfig {
             modules: Default::default(),
         };
         let consensus = ServerConfigConsensus {
+            hash: sha256::Hash::all_zeros(),
```

Comment:
That's an ugly hack, either make it an `Option` or move the cached hash out (it can be regenerated anyway)